conference/react-js/src/containers/ManageUsers/ManageUsers.jsx

```diff
@@ -35,40 +35,27 @@ class ManageUser extends PureComponent {
   render() {
     const { users } = this.props;
     return (
-      <div className="tabs-layout">
-        <div className="tabs-wrapper">
-          <ul className="tabs-list">
-            <li className="tabs-list__item">
-              <a
-                className="tabs-list__anchor tabs-list__anchor_active"
-              >
-           Manage Users
-              </a>
-            </li>
-          </ul>
-          <div className="tabs-container">
-            <div className="my-talks__header">
-              <button className="btn my-talks__button">
-             Add New User
-              </button>
+      <div className="tabs-container">
+        <div className="my-talks__header">
+          <button className="btn my-talks__button">
+          Add New User
```

Comment:
Linter did not fix it. But, i think, You should use more indents when write text